server/api/slack.py

```diff
@@ -51,8 +51,9 @@ def notify_user(message, slack_id):
     """
     return slack_client.api_call(
         "chat.postMessage",
-        channel=slack_id,
+        channel="#general",
         text=message,
+        user=slack_id,
```

Comment:
@dkam26 This parameter does not apply here. The `user` parameter is only relevant when we are sending ephemeral messages using `chat.postEphemeral`. In essence, this `notify_user` method doesn't need to be modified. Please take a look at the docs: 

https://api.slack.com/methods/chat.postMessage
https://api.slack.com/methods/chat.postEphemeral

Reply:
Hey guys,I managed to change the slack notification. It will be via the slackbot as shown in the screenshot below.
![Screen Shot 2019-04-24 at 5 27 51 PM](https://user-images.githubusercontent.com/29740871/56667368-6371e380-66b6-11e9-8fe1-2829f388b628.png)

Reply:
Great @dkam26 . This is more effective by utilizing the Slackbot. Thanks.